traffic_manage/traffic_manage/settings.py

```
@@ -0,0 +1,119 @@
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/6.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-6x-=zvzy20qlwoka3%5t1*^bq!=tgte(8u#2_u9hfqf6=uot@q'
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)